scripts/prepare_tau_srir_db.py

```diff
@@ -0,0 +1,123 @@
+import os
+import pickle
+import scipy
+import mat73
+import numpy as np
+
+from utils import parse_args, get_y
+from pathlib import Path
+
+ROOM_NAMES = ['bomb_shelter', 'gym', 'pb132', 'pc226', 'sa203', 'sc203', 'se203', 'tb103', 'tc352']
+OUTPATH = 'room_scaper/data/rir_databases/TAU_SRIR_DB'
+
+def main():
+    args = parse_args()
+
+    PATH_TO_RIRs = args.path
+
+    measinfo = scipy.io.loadmat(os.path.join(PATH_TO_RIRs,'measinfo.mat'))['measinfo']
+    rirdata = scipy.io.loadmat(os.path.join(PATH_TO_RIRs,'rirdata.mat'))['rirdata']['room'][0][0]
+
+    # get the trajectories for each room 
+    for iroom, room in enumerate(ROOM_NAMES):
+
+        #################################
+        #################################
+        #################################
+        # work first with the rir metadata
+        #################################
+        #################################
+        #################################
+
+        trajs_data = rirdata[iroom][0][2]
+
+        trajs_list_doa = []
+        for traj in trajs_data:
+
+            heights_data = traj
+
+            heights_list = []
+            for height in heights_data:
+
+                heights_list.append(height[0])
+
+            trajs_list_doa.append(heights_list)
+
+        room_out_path_meta = os.path.join(OUTPATH, room, 'metadata')
+        Path(room_out_path_meta).mkdir(parents=True, exist_ok=True)
+
+        with open(os.path.join(room_out_path_meta, 'doa_xyz.pkl'), 'wb') as outp:
+            pickle.dump(trajs_list_doa, outp, pickle.HIGHEST_PROTOCOL)
+
+        #################################
+        #################################
+        # now add the distance information
```

Comment:
@ChrisIck I know you and Rupali have a better way of approximating the distance information for the TAU-SRIR database. Could you incorporate it in this PR?

Reply:
sgtm, I'll add code this week once HPC is back up